docs/sql/statements/vacuum.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: docu
+title: Vacuum
+selected: Documentation/SQL/Vacuum
+expanded: SQL
+railroad: statements/vacuum.js
+---
+
+The `VACUUM` statement garbage collects DuckDB.
```

Comment:
The `VACUUM` statement currently does not garbage collect anything. It was initially added as a nop operation for Postgres compatibility - but now `VACUUM ANALYZE` can be used to recompute statistics if they have gone out of date due to deletions or updates made to the table.

We do plan to add a working `VACUUM` in the future, however.